docs/just.md

```diff
@@ -0,0 +1,37 @@
+# Running `just` commands
+
+We have previously used [make](https://www.gnu.org/software/make/) for easily running some playbook commands (e.g. `make roles` which triggers [`ansible-galaxy`](https://docs.ansible.com/ansible/latest/cli/ansible-galaxy.html)). Our [`Makefile`](../Makefile) is still around, and you can still run these commands.
+
+In addition, we have added support for running commands via [`just`](https://github.com/casey/just) - a more modern command-runner alternative to `make`. It can be used to invoke `ansible-playbook` commands with less typing.
+
+The `just` utility executes shortcut commands (called as "recipes"), which ultimately run `ansible-galaxy`. The targets of the recipes are defined in [`justfile`](../justfile). It defines commands, most of which are not available on our `Makefile`.
+
+For some recipes such as `just update`, our `justfile` recommends to install [agru](https://github.com/etkecc/agru#where-to-get) to speed up the process.
+
+Here are some examples of shortcuts:
+
+| Shortcut                                      | Effect                                                                                                         |
+|-----------------------------------------------|----------------------------------------------------------------------------------------------------------------|
+| `just roles`                                  | Run `make roles` to install the necessary Ansible roles pinned in [`requirements.yml`](../requirements.yml)    |
```

Comment:
This makes it sound like `just roles` would invoke `make roles`, which does some things.

None of our `justfile`-defined recipes invoke `make`.

`just roles` may also use `agru` to install roles (if available) and will fall back to `ansible-galaxy` if not.

While `make roles` exists, it's more simplistic - it only uses `ansible-galaxy`. While it can probably be extended, the `Makefile` is not really maintained (by most of us). The goal is to only use `just`, but.. given that `just` is not very ubiquitous, we cannot pull this off yet and have a minimal old `Makefile` still around.

Reply:
Right, it is my misunderstanding. Fixing…